pkg/kubelet/dockershim/cm/container_manager_linux.go

```diff
@@ -38,10 +38,10 @@ import (
 
 const (
 	// The percent of the machine memory capacity.
-	dockerMemoryLimitThresholdPercent = kubecm.DockerMemoryLimitThresholdPercent
```

Comment:
Maybe move over the whole original comment?

Reply:
Fixed, thanks a lot.